### ISSUE TITLE
Is ie11 support broke?

### ISSUE THREAD

**Issue by dvr707** [grommet/grommet#5576]
Visiting https://v2.grommet.io/ in internet explorer 11 gives a white screen

### Steps to Reproduce

Visit https://v2.grommet.io/
![image](https://user-images.githubusercontent.com/24251723/132370604-3a5dc194-e471-459a-b300-f44fc6d85d4f.png)

### Your Environment

![image](https://user-images.githubusercontent.com/24251723/132371031-6a46fe3b-cee5-4322-b173-2dc395fb9f0e.png)

**Comment by jcfilben** [comment]
Hi @dvr707 thanks for filling the issue. I tested and it does look like the grommet site is no longer working on Internet Explorer.

Microsoft dropped support for Internet Explorer in 2016, will officially retire it in 2022, and plans to entirely shutdown the browser in 2025. Microsoft strongly suggests its users switch to Microsoft Edge due to both security issues and compatibility problems with modern web development practices in Internet Explorer. I tested the site in Microsoft Edge and the site is working as expected.

Since this is the case, I don't know if supporting Internet Explorer is something we want to invest time in. That being said, we always welcome contributions from the community and if someone is particularly interested in this issue they are more than welcome to take a stab at it.
